hw1/01-selectors.md

```diff
@@ -2,6 +2,36 @@
 
 1. plate
 2. bento
-3.
-4.
-5. ...
+3. #fancy
+4. plate apple
+5. #fancy pickle
+6. .small
+7. orange.small
+8. bento orange.small
+
+9. plate, bento
+10. \*
```

Comment:
i think it should be * only. 
I tried "\\*" that didn't work.

Reply:
For Question 10, it is essential to include a forward-slash `/` before the asterisk `*` symbol to ensure it is displayed correctly. When you access your hw1 repository and examine the 01-selector.md file, you will notice that the `*` symbol is not visible in your answer for Question 10. By default, GitHub presents all .md files in the markdown view. To confirm that you have inserted an asterisk, you would need to click the 'Raw' button to view the markdown source code.